server/controllers/deleteProduct.js

```diff
@@ -0,0 +1,10 @@
+const { deleteProductQuery } = require('../database/queries');
+
+const deleteProduct = (req, res) => {
+  const { productId } = req.params;
+  deleteProductQuery(productId)
+    .then(() => res.redirect('/UserProducts'))
```

Comment:
Don't handle the redirect at the backend, you should send a response with the success of the operation and at the front end they handle the response proberly